.cicd/scripts/srw_metric_example.sh

```diff
@@ -3,22 +3,49 @@
 # The goal of this script is to provide an example of performing Indy-Severe-Weather test run and compare results to reference with
 # Skill score index that is calculated by MET Stat-Analysis Tools
 #
-# Required:
+#
+# Required (these options are set in the Jenkins env:
 #    WORKSPACE=</full/path/to/ufs-srweather-app>
 #    SRW_PLATFORM=<supported_platform_host>
 #    SRW_COMPILER=<intel|gnu>
+#    SRW_PROJECT=<platform_account>
 #
 # Optional:
-[[ -n ${SRW_PROJECT} ]] || SRW_PROJECT="no_account"
+#[[ -n ${SRW_PROJECT} ]] || SRW_PROJECT="no_account"
 [[ -n ${FORGIVE_CONDA} ]] || FORGIVE_CONDA=true
 set -e -u -x
 
+BUILD_OPT=false
+RUN_WE2E_OPT=false
+RUN_STAT_ANLY_OPT=false
+
+if [[ $# -eq 0 ]]; then
+    BUILD_OPT=true
+    RUN_WE2E_OPTi=true
```

Comment:
Should this be RUN_WE2E_OPT?

```suggestion
    RUN_WE2E_OPT=true
```

Reply:
resolved, thanks!